src/Cohesity.Powershell/Scripts/Utility/Update-CohesityCmdletConfig.ps1

```diff
@@ -0,0 +1,26 @@
+function Update-CohesityCmdletConfig {
```

Comment:
Why do you need update cmdletconfig again.. cant we do this via set ?

Reply:
We have made it modular. The name could be confusing here, had to adhere to the acceptable powershell verbs. Update-CohesityCmdletConfig is actually doing the job of upgrade. Set-CohesityCmdletConfig only sets the configuration flags.

Reply:
It's ok, having 1 commandlet for setting/updating configurations should be ok. Overwrite when one sets the configs.

Reply:
Upgrade cmdlet config for modular, as agreed.